environment.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - tensorboard
```

Comment:
I remember you mentioned that some tf version is not compatible. Can you please update tf version here too?